packages/date-time/src/components/WeeklyDayPicker/examples/WeeklyDayPicker.Inline.Expandable.Example.tsx

```diff
@@ -0,0 +1,125 @@
+import * as React from 'react';
+import { WeeklyDayPicker, DayOfWeek, addDays, IWeeklyDayPickerProps } from '@uifabric/date-time';
+
+import * as styles from './WeeklyDayPicker.Example.scss';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
+
+const DayPickerStrings = {
+  months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+  shortDays: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+  goToToday: 'Go to today',
+  weekNumberFormatString: 'Week number {0}',
+  prevMonthAriaLabel: 'Previous month',
+  nextMonthAriaLabel: 'Next month',
+  prevYearAriaLabel: 'Previous year',
+  nextYearAriaLabel: 'Next year',
+  prevYearRangeAriaLabel: 'Previous year range',
+  nextYearRangeAriaLabel: 'Next year range',
+  closeButtonAriaLabel: 'Close'
+};
+
+export interface IWeeklyDayPickerInlineExpandableExampleState {
+  selectedDate?: Date;
+  expanded?: boolean;
+}
+
+export interface IWeeklyDayPickerInlineExpandableExampleProps extends Omit<IWeeklyDayPickerProps, 'strings'> {
+  showExpandButton?: boolean;
+  showNavigateButtons?: boolean;
+}
+
+export class WeeklyDayPickerInlineExpandableExample extends React.Component<
```

Comment:
Thanks for adding an example! 

Has there been more design consideration given to this? I see that there's just an expand/collapse button - can we consider a chevron button to bring up & down the full month view?
![image](https://user-images.githubusercontent.com/4161791/65639412-f3d18e80-df9c-11e9-93d9-6489c964d220.png)

Reply:
We plan on using this as a small control in a larger OWA component- there will be a button in a menu there used to expand/collapse. I see this more as a smaller piece, much like CalendarDayGrid is a smaller component. If someone wanted to use a more complete wrapper component around this that just had the header with a chevron arrow, that'd be fine, but I think we'd create a separate component for that.